packages/compiler/lib/intrinsics.tsp

```diff
@@ -1,4 +1,5 @@
-import "../dist/src/lib/intrinsic-decorators.js";
+import "../dist/src/lib/intrinsic/tsp-index.js";
+import "./prototypes.tsp"; // TODO: figure out what to do about this organization
```

Comment:
Any opinions on how to organize intrinsics

Reply:
could:
1. Keep it as it is `intrinsic.tsp`(entrypoint) and `prototypes.tsp`
2. `intrinsics/main.tsp` and `intrinsics/prototypes.tsp`

Reply:
I like the current split.